OpenRA.Mods.Common/Traits/World/Selection.cs

```diff
@@ -237,5 +244,22 @@ public void Deserialize(World world, List<MiniYamlNode> data)
 				}
 			}
 		}
+
+		List<MiniYamlNode> IGameSaveTraitData.IssueTraitData(Actor self)
```

Comment:
These wrappers aren't needed - just rename `public List<MiniYamlNode> Serialize()` to `List<MiniYamlNode> IGameSaveTraitData.IssueTraitData(Actor self)`.